contrib/godot/patches/miniupnpc.patch

```diff
@@ -0,0 +1,29 @@
+API changed in miniupnpc 2.2.8
+
+Patch-Source: https://git.alpinelinux.org/aports/tree/testing/godot/miniupnpc.patch
```

Comment:
these have to point to something stable and not a url that will break in 2 weeks, like https://gitlab.alpinelinux.org/alpine/aports/-/blob/8dd3d8861eff2b4b5171d861e9cde50a92adb93b/testing/godot/miniupnpc.patch

Reply:
cgit also has a way to get a permalink (https://git.alpinelinux.org/aports/tree/testing/godot/miniupnpc.patch?id=44b273d76190945ed980cad6b16276b88017a859; note the `?id=commit`) but i don't really see a point in linking to it so i didn't use it in the example :P

Reply:
there's also another one to fix above and below